about/package-scope.md

```diff
@@ -18,21 +18,257 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
+
+These packages should have a clear scientific application. Wrappers must provide
+significant added value to the scientific ecosystem be it in data handling, or
+improved installation processes for Python users.
+
+We strongly encourage submissions that wrap tools that are open-source with
+an OSI-approved license. Exceptions will be evaluated case-by-case,
+considering whether open-source options exist.
+
+<!-- TODO: need an example for this category -->
+* Examples: We don't have a package in this category yet - *Could be your package?*
+
+### Workflow automation & versioning
+Tools that automate and link together workflows and as such support
+reproducible workflows. These
+tools may include build systems and tools to manage continuous integration.
+This also includes tools that support version control.
+
+<!-- TODO: marting git package is a good example but it's needs to be sunsetted as it's no longer maintained. hamilton is just starting review but would fit
+well here  - the link below for it is a presubmission not a review   -->
+* Examples: [Hamilton - currently under review](https://github.com/pyOpenSci/software-submission/issues/74), martin's git package (that is no longer maintained.)
+
+### Citation management and bibliometrics:
+
+Tools that facilitate managing references, such as for writing manuscripts,
+creating CVs or otherwise attributing scientific contributions, or accessing,
+manipulating or otherwise working with bibliometric data. (Example: [Example from rOpenSci - RefManageR](https://github.com/ropensci/software-review/issues/119))
+
+### Data visualization & analysis
+These are packages that enhance a scientists experience visualizing and
+analyzing data.
+
+* Examples: [PyGMT - (also spatial and data munging)](https://github.com/pyOpenSci/software-submission/issues/43),
+
+### Database software bindings
+
+ Bindings and wrappers for database APIs.
+
+ * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
+
+
+## Domain areas
+
+In addition, our scope includes focused domain areas. These areas are based on
+partnerships that we form with communities and also expertise that we hold
+within our organization. As we develop [new community partnerships](/partners/scientific-communities) and grow,
+we will expand this list.
+
+### Geospatial
+
+Packages focused on the retrieval, manipulation, and analysis of spatial data.
+
+* Examples: [PyGmt](https://github.com/pyOpenSci/software-submission/issues/43),
+[Moving Pandas ](https://github.com/pyOpenSci/software-submission/issues/18)
+
+### Pangeo
+
+We have a [partnership with Pangeo](../partners/pangeo). Often times packages submitted as a part of that partnership are also in the geospatial domain.
+
+* Examples: [xclim - under review now](https://github.com/pyOpenSci/software-submission/issues/73)
+
+### Education
+
+Packages to aid with instruction.
+<!--TODO -  Earthpy fit in this category but it also needs to be sunsetted -->
```

Comment:
Examples: [pyrolite](https://github.com/pyOpenSci/software-submission/issues/20)

Reply:
oh they did tag their package as being education focused too. it's not explicitly used for teaching but should we add it here?

Reply:
added!